internal/doc/main.go

```diff
@@ -90,7 +90,9 @@ func generate(dir string) error {
 		if err != nil {
 			return err
 		}
-		defer f.Close()
+		defer func() {
```

Comment:
internal/doc/main: Ignore file close errors 
ce745fe4f79a74549b369f29b19815a35ceae6fb

Missing lint details in commit

This is not important but Is there a reason we did not do a "*: ignore file close errors" and include this like that rather then per file separation?

Reply:
This is a good question. I was wondering about the impact of us not taking those potential errors into account so I took at look at them one by one. It's unlikely than any of those would ever happen but they could, so we have to consider what to do in each case. For most cases, the error are extremely unlikely or would happen so late that it would not make sense to act on it. But each case is a little bit different thus the different commits.